yjit/src/asm/mod.rs

```diff
@@ -26,6 +26,12 @@ impl From<*mut u8> for CodePtr {
     }
 }
 
+impl CodePtr {
+    pub fn null() -> Self {
+        return CodePtr(0 as *const u8);
+    }
+}
```

Comment:
I've intentionally not implemented this and have been using `Option<CodePtr>` where this could be null.

Reply:
Okay. Changed.